src/commonMain/kotlin/it/krzeminski/snakeyaml/engine/kmp/constructor/json/ConstructYamlBinary.kt

```diff
@@ -25,7 +25,7 @@ class ConstructYamlBinary : ConstructScalar() {
         // Ignore white spaces for base64 encoded scalar
         // TODO decodeBase64() doesn't seem to require removing whitespace - perhaps this can be removed?
         //      I'm leaving it in because I'm not confident about edge case test coverage for whitespaces.
-        val noWhiteSpaces = constructScalar(node).replace("\\s".toRegex(), "")
+        val noWhiteSpaces = constructScalar(node).replace("""\s""".toRegex(), "")
```

Comment:
@suggestion: We could extract this Regex into a constant to reuse between invocations (like it is done for most of other Regex patterns)